src/compounds/Messages/Messages.jsx

```diff
@@ -1,25 +1,88 @@
-import React from 'react'
-import { Card } from 'react-bootstrap'
+import React, { useState } from 'react'
 import PropTypes from 'prop-types'
+import { Card, Collapse } from 'react-bootstrap'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { timeSince, formatBytes } from '../../resources/utilityFunctions'
+import Link from '../../components/Link/Link'
 import Title from '../../components/Title/Title'
 import TextBox from '../../components/TextBox/TextBox'
 
-const Messages = ({ addClass, messages }) => (
+const MessageCardBody = ({messages, message, idx}) => {
+  const { avatar, body, id, name, attachments, dateAndTime } = message
+  return (
+    <Card key={id} className={`${idx === (messages.length - 1) ? '' : 'pb-4'} pt-4 flex-row border-0 border-top`}>
```

Comment:
we don't need to pass the entire messages array and the index, just to compare them here.

you could pass in a variable that equates to true or false and set the class that way. 

``` js
<MessageCardBody message={message} isLatestMessage={idx === (messages.length - 1)} />

className={`${isLatestMessage ? '' : 'pb-4'}
```